### DISCUSSION TITLE
Examples of using `--listen`

### DISCUSSION THREAD

**Discussion by grimnight** [junegunn/fzf#3964] [Q&A]
There doesn't seem to be example script of using `--listen`.
I would have assumed that atleast [ADVANCED] would have one.

Can anyone provide examples or scripts of how they are using `--listen`?

[ADVANCED]: https://github.com/junegunn/fzf/blob/master/ADVANCED.md

**Comment by LangLangBart** [comment]
The `CHANGELOG.md`[^1] has some examples.

The `--listen` flag allows any action that can be specified in `--bind` to be invoked.

> Can anyone provide examples or scripts of how they are using --listen?

There are some examples posted in older Discussions[^2] and Issues[^3]; searching for `--listen` should have surfaced them.

[^1]: [fzf/CHANGELOG.md at master · junegunn/fzf · GitHub](https://github.com/junegunn/fzf/blob/master/CHANGELOG.md)
[^2]: [[Code Snippet] Preview window with updating plot · junegunn/fzf · Discussion #3686](https://github.com/junegunn/fzf/discussions/3686)
[^3]: [Temporary "Loading..." in the header · Issue #3551 · junegunn/fzf](https://github.com/junegunn/fzf/issues/3551)
